### PR TITLE
Remove error log in load by aggregate

### PR DESCRIPTION
**Pull request by perma**
Only error should be returned so the client can choose whether to log or not

### CODE REVIEW

File: eventsource/stores/dynamodbstore/store.go [hunk 1]
```diff
@@ -43,11 +44,7 @@ func (store *store) LoadByAggregate(ctx context.Context, aggregateID string) (re
 
 	output, err := store.db.QueryWithContext(ctx, &input)
 	if err != nil {
-		log.
-			WithField("input", input).
-			WithField("error", err).
-			Error("Couldn't scan pages")
-		err = errors.Wrap(err, "couldn't scan pages")
+		err = fmt.Errorf("couldn't scan pages (input=%+v): %w", input, err)
```
Comment (karlssonerik) [inline eventsource/stores/dynamodbstore/store.go:47]: Could you remove the log and change from wrros.Wrap to fmt.Errorf for the other methods in the file also?

**Review by karlssonerik — CHANGES_REQUESTED** [review]

File: eventsource/stores/dynamodbstore/store.go (hunk 1 above)
Comment (Ekberg) [inline eventsource/stores/dynamodbstore/store.go:47]: We should probably do the same in v2. Are you using v1?

Comment (perma) [reply eventsource/stores/dynamodbstore/store.go:45]: Done.

Comment (perma) [reply eventsource/stores/dynamodbstore/store.go:45]: Turned out we were using v2. :)
Fixed that one as well.

**Review by Ekberg — APPROVED** [review]

**Review by karlssonerik — APPROVED** [review]

**Pull request merged**